src/views/statutory/PositionsList.vue

```diff
@@ -200,7 +200,7 @@ export default {
       return this.$route.params.prefix
     },
     taskDescription () {
-      if (!this.selectedPosition.body_id) {
+      if (!this.selectedPosition.body_id || !this.selectedPosition.task_description || this.selectedPosition.task_description == '') {
```

Comment:
Maybe two seperate messages? One that is 'A body for this position has not been set by JC.' and the other being the one already present

Reply:
Hm for users this does not really matter I think. The only thing that you achieve with this is that people are going to complain to JC about things